conn.c

```diff
@@ -33,13 +33,13 @@ make_conn(int fd, char start_state, tube use, tube watch)
     job j;
     Conn *c;
 
-    c = new(Conn);
-    if (!c) return twarn("OOM"), NULL;
+    c = (Conn *)new(Conn);
```

Comment:
The changes in this file seem unnecessary. The return value of macro `new` is `void*`, which doesn't require a cast to convert to another type. Same goes for testserv.c.

If it does turn out to be necessary, let's do it in a separate pull request.